packages/authentication/README.md

```diff
@@ -54,7 +54,7 @@ import {BasicStrategy} from 'passport-http';
 
 export class MyAuthStrategyProvider implements Provider<Strategy> {
   constructor(
-    @inject(BindingKeys.Authentication.METADATA)
+    @inject(AuthenticationBindings.METADATA)
```

Comment:
Nit:

I'd prefer not to have the `AuthenticationBindings` type of object in my components or apps since its just clutter. Can we structure these in a way where you just import the `METADATA` constant?